cmd/ci-operator-configresolver/main.go

```diff
@@ -89,10 +88,10 @@ func resolveConfig(o *options) http.HandlerFunc {
 			Variant: variant,
 		}
 
-		config, ok := o.configs[info.Basename()]
-		if !ok {
+		config, err := agent.GetConfig(info)
+		if err != nil {
 			w.WriteHeader(http.StatusBadRequest)
-			w.Write([]byte("config does not exist"))
+			fmt.Fprintf(w, "failed to get config: %v", err)
```

Comment:
That error is more of a bad request issue (like someone typing `opnshift` for the org instead of `openshift`). I'll add a warning log for it just in case there is a problem and we need to debug. The `json.Unmarshal` error on the otherhand I'll make an Error level log.